static/src/javascripts/projects/common/modules/experiments/tests/headlines.js

```diff
@@ -0,0 +1,58 @@
+define([
+    'common/utils/_',
+    'bonzo',
+    'fastdom',
+    'qwery'
+], function (
+    _,
+    bonzo,
+    fastdom,
+    qwery
+) {
+    return function (n) {
+        this.id = 'Headline' + n;
+        this.start = '2015-04-9';
+        this.expiry = '2015-06-10';
+        this.author = 'Robert Berry';
+        this.description = 'A/B test for headline number ' + n;
+        this.audience = 0.01;
+        this.audienceOffset = 0.01 * n;
```

Comment:
We have other tests running in this range at the moment. I'd suggest moving, unless you are not worried about them conflicting with your data?